j2v/generation/generator.py

```diff
@@ -50,10 +51,14 @@ def collect_all_paths(self, current_dict, current_path=None, current_view=None,
             if type(key) != str:
                 continue
             if is_primitive(value) or value is None:
-                self.__add_dimension(current_path, current_view, key, value)
+                if group_label is not None and doublequote(group_label) in current_path.split(":"):
```

Comment:
@ruthmd that's an incorrect hack. We cannot just search in the whole path for the parent_key.

Reply:
Yes! Thanks for that.